policy/modules/contrib/virt.te

```diff
@@ -2307,6 +2309,9 @@ storage_manage_fixed_disk(virtqemud_t)
 sysnet_exec_ifconfig(virtqemud_t)
 sysnet_manage_config(virtqemud_t)
 
+term_use_generic_ptys(virtqemud_t)
+term_use_ptmx(virtqemud_t)
+
```

Comment:
Is there a specific use case when these terms are needed? Is it worth having it behind a boolean?